service/routes.py

```diff
@@ -0,0 +1,47 @@
+import time
+import logging
+
+from flask import request
+
+from service.attack_surface import AttackSurfaceService
+from service.consts import ApiConsts
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+logging.basicConfig(filename='app.log', filemode='w')
+
+
+def configure_routes(app, input_file_path):
+    attack_surface = AttackSurfaceService(
+        input_file_path=input_file_path,
+    )
+
+    @app.route('/api/v1/stats', methods=['GET'])
+    def stats():
+        logger.info(f'{time.time()} stats endpoint was called')
+        attack_surface.incr_req_counter()
+        requests_count = attack_surface.request_count
+        avg_process_time = \
+            attack_surface.cumulative_process_time / requests_count \
+            if requests_count > 0 else 0
+
+        stats_dict = {
+            'request_count': requests_count,
+            'vm_count': attack_surface.vms_number,
+            'average_request_time': round(
+                avg_process_time,
+                ApiConsts.ROUND_DECIMAL_NUMBER,
+            ),
+        }
+        return stats_dict
+
+    @app.route('/api/v1/attack', methods=['GET'])
+    def attack():
+        logger.info(f'{time.time()} - attack endpoint was called')
+        attack_surface.incr_req_counter()
+        vm_id = request.args.get('vm_id')
```

Comment:
what happens if a different argument is passed? how do you handle improper input?